File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--synchronizing-a-custom-task-pane-with-a-ribbon-button_2.cs
        private void ThisAddIn_Startup(object sender, System.EventArgs e)
        {
            taskPaneControl1 = new TaskPaneControl();
            taskPaneValue = this.CustomTaskPanes.Add(
                taskPaneControl1, "MyCustomTaskPane");
            taskPaneValue.VisibleChanged +=
                new EventHandler(taskPaneValue_VisibleChanged);
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--synchronizing-a-custom-task-pane-with-a-ribbon-button_2.vb
    Private Sub ThisAddIn_Startup(ByVal sender As Object, _
        ByVal e As System.EventArgs) Handles Me.Startup

        taskPaneControl1 = New TaskPaneControl()
        taskPaneValue = Me.CustomTaskPanes.Add( _
            taskPaneControl1, "MyCustomTaskPane")
    End Sub